README.md

```diff
@@ -192,6 +194,33 @@ us to build a hierarchical dependency tree of all puzzles, like
 for example. Technically, of course, you can abuse the system
 and put a dummy `#1` marker everywhere.
 
+### Multiline examples
+
+```xml
+<!-- 
+ ~ if comment should be started and closed by special symbols, then place them in
```

Comment:
It can be used to decorate text, not required, but for there is no restrictions about the prefix, it should be just equal for all text. So it is just an example

Reply:
@pnatashap, maybe we can add this to the documentation? I mean, we can state that the "prefix" is a symbol that can be used to decorate text; it isn't required. And provide some examples. 

Moreover, I would suggest adding some headers or small - one-sentence descriptions to these code examples:

1. "...Prefix definition... Here we ignore the prefix, and only one puzzle will be created."
2. "We can also use the `\` symbol to connect multiline puzzles."

What do you think?